### ISSUE TITLE
Can't save job listings (without location) since Freebase API is no longer available

### ISSUE THREAD

**Issue by anarchivist** [code4lib/shortimer#57]
The autocomplete behavior provided by the Freebase API is no longer working for locations (as well as other tags). Without a location, jobs in the curation queue can't be saved or published.

**Comment by edsu** [comment]
Yes, unfortunately the site is in a holding pattern until #38 is resolved. I've done a bit of work moving existing data over to wikidata but there is more work to do, and I'm not 100% sure when I will get to it. Hopefully in the next month.
